js_modules/dagster-ui/packages/ui-core/src/assets/catalog/AssetCatalogTableV2.tsx

```diff
@@ -203,17 +258,47 @@ export const AssetCatalogTableV2 = React.memo(
 
 AssetCatalogTableV2.displayName = 'AssetCatalogTableV2';
 
+const SORT_ITEMS = [
+  {
+    key: 'materialization_asc' as const,
+    text: 'Materialization (asc)',
+  },
+  {
+    key: 'materialization_desc' as const,
+    text: 'Materialization (desc)',
+  },
+  {
+    key: 'key_asc' as const,
+    text: 'Asset key (asc)',
+  },
+  {
+    key: 'key_desc' as const,
+    text: 'Asset key (desc)',
+  },
+];
```

Comment:
Use an enum for the keys so that the types below can be cleaned up a bit?

Reply:
I'll just make a type for `(typeof SORT_ITEMS)[number]` since I use that a lot

Reply:
(I like having the types come from this array rather than having to define both an enum and the type for this array).

Reply:
It feels like a good since source of truth